src/app-shared/core/service/account/account.service.ts

```diff
@@ -15,10 +15,19 @@ import { IJsonConverterService } from '../json-converter/json-converter.service'
 import { IWebRequestService, RequestContentType } from '../web-request/web-request.interface';
 
 export interface IAccountCredentials {
-  username: string;
+  email: string;
   password: string;
 }
 
+export interface IAccountForgotPasswordReq {
```

Comment:
I do not like naming of interfaces or classes that have short notation. Use `Request` vs `Req` instead.

Reply:
Done, in MR #111